hal/src/stm32f2xx/i2c_hal.c

```diff
@@ -237,6 +237,8 @@ int HAL_I2C_Init(HAL_I2C_Interface i2c, const HAL_I2C_Config* config)
     if(i2c == HAL_I2C_INTERFACE1)
     {
         i2cMap[i2c] = &I2C_MAP[I2C1_D0_D1];
+        HAL_Pin_Mode(i2cMap[i2c]->I2C_SDA_Pin, INPUT);
```

Comment:
Looks weird that we are only initializing D0 and D1 and not not any pins from the other interfaces. Do we need to do this in init. or can we rely on `HAL_I2C_Begin()` to simply initialize pins as AF?